hub/instruction_handling/call/precompiles/common/failure_KTH.tex

```diff
@@ -14,8 +14,9 @@
 \instRipemd{},
 \instIdentity{},
 \instEcadd{},
-\instEcmul{} or
-\instEcpairing{} precompiles.
+\instEcmul{},
+\instEcpairing{} or 
+\instPVerify{} precompiles.
```

Comment:
IIRC all precompiles introduced in Cancun / Prague, as well as the Osaka ones, can raise the `FKTH` flag. So the documentation was missing POINTEVAL / BLS_XXX too.